src/bidiMapper/domains/network/NetworkProcessor.ts

```diff
@@ -14,6 +14,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import {URLPattern} from 'urlpattern-polyfill';
```

Comment:
I think we should try to load it only if its not there, like we did with `uuid`.

Reply:
Done